### PR TITLE
Create cquarte1.md

### PR DESCRIPTION
**Pull request by cquarte1**
https://raw.githack.com/cquarte1/cquarter.github.io/CarlosVI/#!pages/vi/profiles/cquarte1.md

### CODE REVIEW

**Comment by lmmrssa** [comment]
@cquarte1 You did not add any markdown syntax. You just added content. This is part of Step 3 which is supposed to teach usage of markdown.

File: pages/vi/profiles/cquarte1.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 **Carlos Quarterman**
-_Albany, GA_
+#Albany, GA#
```
Comment (lmmrssa) [inline pages/vi/profiles/cquarte1.md]: \# is supposed to be only added at front. not at both end.

Comment (cquarte1) [reply pages/vi/profiles/cquarte1.md]: Just change resolve the issue

**Comment by cquarte1** [comment]
> Hi @cquarte1, your forked repository appears to be named incorrectly. Also, please use at least 5 different types of Markdown elements for your profile so that you can get an idea of how it works.

**Pull request closed without merging**